Add a definition for the macro __ELF_NATIVE_CLASS which depends
on the native arch's word size.

--- src/mongo/util/stacktrace_posix.cpp	2015-08-24 02:39:52.000000000 +0200
+++ src/mongo/util/stacktrace_posix.cpp	2015-08-26 18:59:52.290831910 +0200
@@ -37,6 +37,15 @@
 #include <string>
 #include <sys/utsname.h>
 
+#if !defined(__GLIBC__)
+#if defined __x86_64__ && !defined __ILP32__
+# define __WORDSIZE	64
+#else
+# define __WORDSIZE	32
+#endif
+#define __ELF_NATIVE_CLASS __WORDSIZE
+#endif
+
 #include "mongo/base/init.h"
 #include "mongo/config.h"
 #include "mongo/db/jsobj.h"
